File: 2088/2088.cs
public class Solution {
    public int CountFertilePyramidsInALand(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(CountFertilePyramidsInALand(root.left), CountFertilePyramidsInALand(root.right));
    }
}

File: 2088/2088.c
int countFertilePyramidsInALand(struct TreeNode* root) {
    if (!root) return 0;
    int l = countFertilePyramidsInALand(root->left);
    int r = countFertilePyramidsInALand(root->right);
    return 1 + (l > r ? l : r);
}